template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazDatalakeConfigProvider.java

```diff
@@ -25,7 +26,8 @@ public boolean isConfigurationNeeded(CmTemplateProcessor cmTemplateProcessor, Te
         return StackType.DATALAKE == source.getStackType()
                 && CMRepositoryVersionUtil.isRazConfigurationSupported(
                         source.getProductDetailsView().getCm().getVersion(), source.getCloudPlatform(), source.getStackType())
-                && source.getGeneralClusterConfigs().isEnableRangerRaz();
+                && source.getGeneralClusterConfigs().isEnableRangerRaz()
+                && TargetPlatform.PAAS.equals(TargetPlatform.getByCrn(source.getDatalakeView().get().getCrn()));
```

Comment:
based on my previous suggestion, you could use CRN field from TemplatePreparationObject